@navikt/core/tokens/figma-api/colors.mjs

```diff
@@ -0,0 +1,66 @@
+import { getLocalColors } from "./api.mjs";
+import Color from "color";
+import { prefix } from "./config.mjs";
+import { colorFrames, colorPageName, colorElement } from "./config.mjs";
+
+const parseName = (name) => {
+  const split = name.toLowerCase().split("/");
+  const group = split.shift();
+  return `${prefix}-${group}-color-${split.join("-")}`;
+};
+
+const getColorRef = (color, colors) => {
+  const globalColors = colors.filter((x) =>
+    x.name.startsWith(`${prefix}-global`)
+  );
+
+  for (const x of globalColors) {
+    if (Color(x.color).string() === Color(color).string()) {
+      return `{${prefix}-${x.name}.value}`;
+    }
+  }
+
+  return color;
+};
+
+export const getColors = async () => {
```

Comment:
Her skjer det mye custom logikk, så dette tenker jeg er et tilfelle hvor vi har mye å hente på å skrive automatiserte tester. For å få til det på en god måte burde vi omstrukturere koden en del også, for å gjøre den mer testbar. Om du vil kan jeg lage ut utkast på det, eller er du gira på å gjøre det sammen, kanskje?

Reply:
Vil gjerne gjøre det sammen! Bare å finne en tid som passer 👌